ocean_lib/assets/test/test_utils.py

```diff
@@ -107,3 +128,16 @@ def test_add_trusted_algorithm_no_compute_service(publisher_ocean_instance, meta
             algorithm_ddo.did,
             publisher_ocean_instance.config.metadata_cache_uri,
         )
+
+
+def test_fail_generate_trusted_algo_dict():
+    """Tests if all the parameters for generating a trusted algorithm dict are None and if teh scope of this function
```

Comment:
you have a typo here: "the" scope and not "teh" scope. I'd also shorten this docstring. E.g. "Tests if generate_trusted_algo_dict throws an AssertionError when all parameters are None."